### DISCUSSION TITLE
Issue receiving messages from Telegram

### DISCUSSION THREAD

**Discussion by mosix76** [arendst/Tasmota#11280] [Q&A] [answered]
Hello Guys,
I was very interested about the possibilty to communicate whith Telegram from a Sonoff with Tasmota on board. So I've compiled the firmware enabling the following paramaters:
```
#define USE_TELEGRAM
#define USE_MQTT_TLS_CA_CERT
```
and then I've uploaded the bin file on the sonoff.

I've created the bot on telegram, and a chat group, where I've added the bot, and I've also setted the parameters on the sonoff. 
It can send messages to the the chat group, but it can't receive any messages from Telegram. 

Below the state:
```
11:36:36 CMD: tmstate 3
11:36:36 RSL: RESULT = {"TmState":{"Send":"ON","Receive":"ON","Echo":"OFF"}}
```

What can I do?

**Comment by barbudor** [comment]
Same for me.
Haven't had time to investigate yet

**Comment by mosix76** [comment, marked as answer]
the answer is here #https://github.com/arendst/Tasmota/issues/8619#issuecomment-711094057
Many thanks JackNJack .
